src/Compilers/Core/Portable/CodeGen/PrivateImplementationDetails.cs

```diff
@@ -576,7 +576,7 @@ public override void Dispatch(Cci.MetadataVisitor visitor)
 
         public Cci.ITypeDefinition ContainingTypeDefinition => _containingType;
 
-        public Cci.TypeMemberVisibility Visibility => Cci.TypeMemberVisibility.Private;
+        public Cci.TypeMemberVisibility Visibility => Cci.TypeMemberVisibility.Assembly;
```

Comment:
Did ILVerify catch a case where we messed up accessibility rules and thus necessitated this change?

Reply:
Yes, we were getting `[M]: Field is not visible. { Offset = 0x0 }` in `NullableValueType_ImplicitConversion_Byte` on the instruction that accesses the field. See diff of commit 2.